sources/packages/backend/apps/load-test-gateway/Dockerfile.dev

```diff
@@ -0,0 +1,35 @@
+# Base Image
+FROM node:18.17.1-alpine3.18
+
+LABEL maintainer="BC GOV"
+
+# Updating container path variable with node bin
+ENV PATH $PATH:/app/node_modules/.bin
+
+WORKDIR /app
+
+# Load test gateway Port
+ENV PORT 3005
+
+# Copying root files to image workdir.
+COPY /*.json /*.js ./
+
+# Installing dependencies
+RUN npm ci
+
+# Copying sources.
+COPY ./apps/load-test-gateway ./apps/load-test-gateway
+COPY ./apps/db-migrations ./apps/db-migrations
```

Comment:
Do we need migrations on this POD? Is there a benefit?

Reply:
It is not required and there is no benefit. The boiler plate code came from queue-consumers and this one missed to be removed.